src/compiler/data.ts

```diff
@@ -18,7 +19,7 @@ import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../type';
  *                 If the encoding contains aggregate value, this will also create
  *                 aggregate table as well.
  */
-export default function(model: Model) {
+export function def(model: Model) {
```

Comment:
I still disagree that `default` is always a bad idea but agree that we should use `compileXXX()`.